src/styles/index.scss

```diff
@@ -1,3 +1,237 @@
+$--blue-color: #00acdc;
+$nav-links-color: #060b35;
+$card-width: 200px;
+$font-family: 'Roboto', sans-serif;
+$main-color: #060b35;
+$secondary-color: #616070;
+$button-bg-color: #00acdc;
+$button-bg-color-hover: #fff;
+$button-text-color: white;
+
 body {
   margin: 0;
+  font-family: $font-family;
+  color: $main-color;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+  &__logo {
+    width: 40px;
+    height: 40px;
+    display: block;
+  }
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  text-transform: uppercase;
+
+  &__list {
+    list-style: none;
+    display: flex;
+    padding: 0;
+    margin: 0;
+    height: 100%;
+  }
+
+  &__item {
+    margin: 0 10px;
+    &:first-child {
+      margin-left: 0;
+    }
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    text-decoration: none;
+    color: $nav-links-color;
+    text-align: center;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+    height: 60px;
+    font-size: 12px;
+    transition-duration: 300ms;
+
+    &:hover {
+      color: $--blue-color;
+    }
+  }
+}
+
+.is-active {
+  position: relative;
+  color: $--blue-color;
+
+  &::after {
+    content: '';
+    display: block;
+    width: 100%;
+    height: 4px;
+    border-radius: 8px;
+    background-color: $--blue-color;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+  }
+}
+
+.catalog {
+  gap: 46px 48px;
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.product-card {
+  width: $card-width;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
```

Comment:
Transforming the whole card on hover may not be the best user experience, as it can be a bit disorienting. It might be better to only change the color or add a shadow on hover.